doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -2,7 +2,573 @@
+to pass a sample period of the temperature to endpoint.
+
+# Adding application
+
+To add an application, proceed as follows:
+
+1. Open the [Kaa admin UI in your browser](http://127.0.0.1:8080) and log in
+as a tenant administrator (by default the user/password is admin/admin123 in sandbox;
+use the correct credentials if you changed the default).
+1. Select **Applications** on the navigation panel on the left side and, in the
+Applications window that opens, click **Add application**.
+1. In the Add application window, enter the application name and then click **Add**.
+For this guide we will use **Trustful** credential service and our application will be
+named "My First Kaa Application"
+
+    ![Admin Console](attach/new_app.png)
+
+After the application has been added, you may log out. We will not be using
+the tenant administrator role in this guide anymore.
+
+# Creating schemas
+
+The application that you have created in [the previous step](#adding-application)
+already includes the default versions of the profile, configuration, notification
+and log schemes ready for use. However, in this sample application, we will use a custom data
+collection and configuration schemas for demonstration purposes.
+
+To create and upload the schemas, proceed as follows:
+
+1. Create the `data-schema.json` file on your PC with the following schema definition:
+
+        {
+            "type": "record",
+            "name": "DataCollection",
+            "namespace": "org.kaaproject.kaa.schema.sample",
+            "fields": [
+                {
+                    "name": "temperature",
+                    "type": "int"
+                }
+            ]
+        }
+1. Create the `configuration-schema.json` file and add the following schema:
+
+        {
+            "type": "record",
+            "name": "Configuration",
+            "namespace": "org.kaaproject.kaa.schema.sample",
+            "fields": [
+                {
+                    "name": "samplePeriod",
+                    "type": "int",
+                    "by_default": "1"
+                }
+            ]
+        }
+
+    `by_default` field defines the default value for sampling period, which is in our example
+    is set to 1 second.
+1. Open the admin UI in your browser and log in as a tenant developer
+(default user/password in sandbox: devuser/devuser123; use the correct credentials if
+you changed the default).
+1. Open the relevant **Log schemas** window (**Applications =>
+My First Kaa Application => Schemas => Log**) and click **Add schema**.
+1. In the **Add log schema** window, enter the name and description of
+the new data collection schema.
+
+    ![Data collection scheme](attach/new_data_schema.png)
+1. Scroll down and use the **Upload from file** function to find the previously
+    created json file with the schema. Alternatively, you can use the
+    [Schema Avro UI]({{root_url}}/Administration-guide/Tenants-and-applications-management/#avro-ui-forms)
```

Comment:
Oh, you said about GH. Yet, it will not work, since it contains `{{root_url}}` which is recognizable only by Jekyll